src/average.h

```diff
@@ -19,7 +19,7 @@ namespace Faunus
       } //!< Root-mean-square
 
       double stdev() const {
-          return std::sqrt( cnt / static_cast<double>(sqsum) - avg()*avg() );
+          return std::sqrt( (sqsum + avg()*avg() - 2*sum*avg()) / static_cast<double>(cnt-1) );
```

Comment:
I think this should be:

`return std::sqrt( (sqsum + cnt*avg()*avg()- 2*sum*avg()) / static_cast<double>(cnt-1) )`